exercises/creating-a-pull-request/ABOUT_ME.md

```diff
@@ -0,0 +1,7 @@
+# About me
+
+## Shakti Prasad Nanda - name
+
+Hi I am **Shakti** !
+*I like experimenting stuffs and curious about stuff that people find mundane.*
```

Comment:
This line does not make sense to me. If unsure about grammar, please use Grammarly or similar tool to proofread your text.

Reply:
I have changed that line to say something else.